packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -182,13 +159,14 @@ export class OdspDocumentService implements IDocumentService {
                 this.epochTracker,
                 // flushCallback
                 async () => {
-                    if (this.currentConnection !== undefined && !this.currentConnection.disposed) {
```

Comment:
how do we ensure that (through some accidental changes) some callback like this does not execute on critical boot path? Will we notice that? Do our bundle size tests capture that?

Reply:
Handled in this PR: https://github.com/microsoft/FluidFramework/pull/13519